.github/workflows/pr-ci.yml

```diff
@@ -150,9 +169,66 @@ jobs:
 
           To run smoketest:
           ```
-          # amd64          
+          # amd64
           CRYOSTAT_IMAGE=${{ env.amd64_image }} sh smoketest.sh
 
           # or arm64
           CRYOSTAT_IMAGE=${{ env.arm64_image }} sh smoketest.sh
           ```
+
+  integration-test:
+    needs: [checkout-branch, push-to-ghcr]
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
+    uses: ./.github/workflows/integrated-test.yml
+    with:
```

Comment:
add `skip-itests` or only run with `build-arch: amd64` since tests run with `arm64` image will surely fail

Reply:
same at https://github.com/aali309/cryostat/blob/cd43ca4aa0389b949287d0de21793d437da49a10/.github/workflows/pr-ci.yml#L197

Reply:
also `checkout-repo` and `checkout-ref`?

Reply:
Last time i tried it dint work, il try it again now

Reply:
Yah, does not work.. I remember had a conversation with @Andrew the other day about this